### PR TITLE
Gamedb: additions & fixes

### PR DESCRIPTION
**Pull request by rufotheone-fr**
### Description of Changes
1269c80
* SLES-52856 "10 Most Wanted"
* SLPM-55014 "Aria - The Origination - Aoi Hoshi no El Cielo"
* SLPM-55282 "Armen Noir"
* SLPM-62320 "2003-nen Kaimaku - Ganbare Kyuukai-ou - Iwayuru Pro Yakyuu desu ne"
* SLPM-62323 "AI Shougi 2003"
* SLPM-62329 "AI Igo 2003"
* SLPM-62330 "AI Mahjong 2003"
* SLPM-65037 "Angelique Trois - Aizouban [Shokai Gentei Package]"
* SLPM-65151 "Akudaikan"

### Rationale behind Changes
Consistent and accurate gamedb

### CODE REVIEW

**Comment by rufotheone-fr** [comment]
2a79318
* SCES-52764 "DJ - Decks & FX - Live Session"
* SCES-55386 "Buzz! Le Plus Malin des Francais"
* SCES-55670 "Don 2 - The Game"
* SLES-52874 "Dark Wind (with Gametrak)" => "Dark Wind [with Gametrak]"
* SLES-53952 "Cue Academy, The (Snooker-Pool-Billiards)" => "Cue Academy - Snooker, Pool, Billiards"
* SLPM-55060 "D.Gray-man - Sousha no Shikaku"
* SLPM-55187 "Canvas 3 - Tanshoku no Pastel"
* SLPM-55221 "beatmania IIDX 16 EMPRESS + PREMIUM BEST (EMPRESS DISC)" => "beatmania IIDX 16 EMPRESS + PREMIUM BEST [EMPRESS DISC]"
* SLPM-55222 "beatmania IIDX 16 EMPRESS + PREMIUM BEST (PREMIUM BEST DISC)" => "beatmania IIDX 16 EMPRESS + PREMIUM BEST [PREMIUM BEST DISC]"
* SLPM-55239 "Death Connection"
* SLPM-55255 "Clover no Kuni no Alice - Wonderful Wonder World"
* SLPM-55259 "Desert Kingdom"
* SLPM-55281 "Clock Zero - Shuuen no Ichibyou"
* SLPM-62036 "Choukousoku Reversi" => "Chou Kousoku Reversi"
* SLPM-62037 "Chou Kousoku Igo"
* SLPM-62039 "Chou Kousoku Shougi"
* SLPM-62140 "Cyber Jansou - Toufuusou-hen"
* SLPM-62146 "Chou Kousoku Mahjong Plus"
* SLPM-65522 "Bakushou!! Jinsei Kaidou - Nova Usagi ga Miteru zo!!"
* SLPM-66922 "D.C. II P.S. - Da Capo II Plus Situation [DX Pack]" => "D.C.II P.S. - Da Capo II - Plus Situation [DX Pack]"
* SLPM-66923 "D.C.II P.S. - Da Capo II - Plus Situation [Disc 2] [DX Pack]"
* SLPM-66924 "D.C. II P.S. - Da Capo II Plus Situation" => "D.C.II P.S. - Da Capo II - Plus Situation"
* SLPM-68521 "Dot Hack Fraegment [Senkou Release-ban]"
* SLPS-20099 "Dream Audition 3"
* SLPS-20266 "Baskelian"
* SLPS-20506 "Daito Giken Koushiki Pachi-Slot Simulator - 24 - Twenty Four"
* SLPS-25073 "Cinema Surfing - Youga Taizen"
* SLPS-25871 "Drastic Killer"
* SLPS-25872 "Daisenryaku - Daitoua Kouboushi - Tora Tora Tora Ware Kishuu ni Seikou Seri"
* SLUS-21917 "Dance Dance Revolution X2 (Bundle)" => "Dance Dance Revolution X2 [Bundle]"

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -23158,7 +23194,13 @@ SLPM-62035:
   name: "Got to Do! Hot Spring Table Tennis"
   region: "NTSC-J"
 SLPM-62036:
-  name: "Choukousoku Reversi"
+  name: "Chou Kousoku Reversi"
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:23197]: Are you sure on this? The internet is pretty unanimous on "Choukousoku" being a single word (same goes for the next 2 entries also.

**Comment by rufotheone-fr** [comment]
Chou means "great" and kousoku relates to "high speed". These are 2
separate words.
http://redump.org/disc/15574/

Le ven. 15 juil. 2022, 12:13, refractionpcsx2 ***@***.***> a
écrit :

**Comment by refractionpcsx2** [comment]
Thanks, I don't 100% trust redump to be correct (they aren't always), and when every single other source on the internet says it's one word, it makes me question it.

**Comment by rufotheone-fr** [comment]
Search for "chou kousoku" a bunch of series got this in their title (which
could be translated as "ultra high speed").

Le ven. 15 juil. 2022, 12:50, refractionpcsx2 ***@***.***> a
écrit :

**Pull request merged**